scripts/generator/graphql_generator/csharp/SDK/ShopifyError.cs.erb

```diff
@@ -16,7 +16,7 @@ namespace <%= namespace %>.SDK {
         }
 
         public enum ErrorType {
-            HTTP, GraphQL, UserError, NativePaymentProcessingError
+            HTTP, GraphQL, UserError, NativePaymentProcessingError, WebCheckoutVerificationError
```

Comment:
I'd be down for getting rid of WebCheckoutValidationError. It seems too specific to web checkout. If we find we need something more specific later we can add something.